samples/complex/RetrievalBot/dotnet/RetrievalBot/Plugins/GetContactsPlugin.cs

```diff
@@ -0,0 +1,69 @@
+using Azure.Core;
+using Microsoft.SemanticKernel;
+using Newtonsoft.Json.Linq;
+
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Net;
+using Microsoft.Graph;
+using Azure.Identity;
+using System.ComponentModel;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Threading.Tasks;
+using Microsoft.Agents.Builder;
+using Microsoft.Agents.Core.Models;
+using Microsoft.Agents.Builder.App;
+using Microsoft.SemanticKernel.Agents;
+
+namespace RetrievalBot.Plugins
+{
+    public class GetContactsPlugin(AgentApplication app)
```

Comment:
This doesn't seem to be in scope for this solution based on the description in the readme